### PR TITLE
[CMM] Implement Narci, Fable Singer

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Narci, Fable Singer]]

File: Mage.Sets/src/mage/cards/n/NarciFableSinger.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    @Override
+    public NarciFableSingerEffect copy() {
+        return new NarciFableSingerEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        int value = (int) getValue("SAGA_MANACOST");
```
Comment (Susucre) [inline Mage.Sets/src/mage/cards/n/NarciFableSinger.java:80]: It is quite hard to track the saga after FinalChapterAbilityResolvesTriggeredAbility trigger.
The targetPointer seem to be alright (there is info on the trigger's rule), but I could not retrieve it in the apply.

Went with the linked setValue/getValue, but there is certaintly a way to make TargetPointer work?

Comment (xenohedron) [reply Mage.Sets/src/mage/cards/n/NarciFableSinger.java:80]: I am not sure what's going on there, so your strategy seems fine.

**Review by xenohedron — APPROVED** [review]

File: Mage.Sets/src/mage/cards/n/NarciFableSinger.java (hunk 1 above)
Comment (JayDi85) [reply Mage.Sets/src/mage/cards/n/NarciFableSinger.java:80]: @Susucre 

Don't use a primitive types in game state values cause it raise exception on nulls -- only object types allows (e.g. `Integer` instead `int`).

Comment (Susucre) [reply Mage.Sets/src/mage/cards/n/NarciFableSinger.java:80]: oh ok, i'll make a pr to fix that one, as it got merged already.

**Pull request merged**